src/Transpiler.ts

```diff
@@ -1164,6 +1240,29 @@ export abstract class LuaTranspiler {
         }
     }
 
+    public transpileIdentifier(identifier: ts.Identifier): string {
```

Comment:
We should probably add a comment with explanation + link to relevant TS docs/code